From 66d546e04f3fdf80159432272308c064b360c10a Mon Sep 17 00:00:00 2001
From: Fei Yang <fei.yang@intel.com>
Date: Sat, 8 Nov 2014 09:41:04 -0800
Subject: [PATCH 1/2] SND:BYT: DMI id list has to be ended with DMI_NONE

The dmi_system_id list has to have an empty id at the end, otherwise
dmi_matches() would keep going until dmi->matches[i].slot is DMI_NONE.

Change-Id: Id67d9325f2783c2c7a7dfe62a881002178ee82bd
Signed-off-by: Fei Yang <fei.yang@intel.com>
Signed-off-by: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
---
 sound/soc/intel/board/byt_cr_board_configs.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sound/soc/intel/board/byt_cr_board_configs.h b/sound/soc/intel/board/byt_cr_board_configs.h
index 6fe5eb19c8dd..37c4cc6ff32a 100644
--- a/sound/soc/intel/board/byt_cr_board_configs.h
+++ b/sound/soc/intel/board/byt_cr_board_configs.h
@@ -69,6 +69,7 @@ static const struct board_config board_configs0[] = {
 		.i2s_port = 2,
 		.speaker_input = 1,
 	},
+	{}
 };
 
 static const struct dmi_system_id dmi_system_ids0[] = {
@@ -88,6 +89,7 @@ static const struct dmi_system_id dmi_system_ids0[] = {
 		},
 		.driver_data = (void *)&board_configs0[RT5640_T100],
 	},
+	{}
 };
 
 /* Configurations - RT5651 */
@@ -117,6 +119,7 @@ static const struct board_config board_configs1[] = {
 		.i2s_port = 0,
 		.speaker_input = 3,
 	},
+	{}
 };
 
 static const struct dmi_system_id dmi_system_ids1[] = {
@@ -128,6 +131,7 @@ static const struct dmi_system_id dmi_system_ids1[] = {
 		},
 		.driver_data = (void *)&board_configs1[RT5651_ANCHOR8],
 	},
+	{}
 };
 
 static const struct mach_codec_link mach_codec_links[] = {
-- 
1.7.9.5

